src/components/HURUmap/Panel/MobileTabPanel/index.js

```diff
@@ -0,0 +1,68 @@
+import { Typography } from "@material-ui/core";
+import Proptypes from "prop-types";
+import React from "react";
+
+import PanelItem from "./PanelItem";
+import useStyles from "./useStyles";
+
+import { ReactComponent as TopIcon } from "@/pesayetu/assets/icons/Component 130 – 1.svg";
+import Tabs from "@/pesayetu/components/Tabs";
+import formatData from "@/pesayetu/utils/formatProfileDataIntoArray";
+import slugify from "@/pesayetu/utils/slugify";
+
+function MobileTabPanel({ data, geography, activeType, ...props }) {
```

Comment:
```suggestion
function MobilePanel({ data, geography, activeType, ...props }) {
```

No need for Tab in name.